src/main/Config.cpp

```diff
@@ -1033,7 +1032,23 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
             }
             else if (item.first == "ENABLE_FLOW_CONTROL_BYTES")
             {
-                ENABLE_FLOW_CONTROL_BYTES = readBool(item);
+                if (readBool(item))
+                {
+                    CLOG_WARNING(
+                        Overlay,
+                        "ENABLE_FLOW_CONTROL_BYTES is now mandatory. This "
+                        "config option will be removed in a future release. "
+                        "Please remove the option from your configuration "
+                        "file.");
+                }
+                else
+                {
+                    CLOG_ERROR(Overlay,
+                               "ENABLE_FLOW_CONTROL_BYTES is now mandatory. "
+                               "Overriding to `true`. This config option will "
+                               "be removed in a future release. Please remove "
+                               "the option from your configuration file.");
+                }
```

Comment:
I think ignoring is fine here. Unlike the BucketListDB config migrations, `ENABLE_FLOW_CONTROL_BYTES` being silently enabled won't break any operators, so we can just warn for a bit then remove the flag.

Reply:
Sounds good! I left this as-is.

Reply:
Tbh I doubt anyone would be affected by the removal of this flag, since it's sort of an advanced low-level config. You can probably just remove it without graceful deprecation, but we do need to make sure that our own infra isn't using the flag somewhere (to avoid crashing on startup during the release).

Reply:
I removed this flag